src/Controls/src/NuGet/README.md

```diff
@@ -0,0 +1,282 @@
+    </TabBar>
+</Shell>
+```
+
+### Collections and Lists
+
+Display lists and collections with CollectionView:
+
+```xml
+<CollectionView ItemsSource="{Binding Items}">
+    <CollectionView.ItemTemplate>
+        <DataTemplate>
+            <Grid Padding="10">
+                <Label Text="{Binding Name}" FontSize="18" />
+            </Grid>
+        </DataTemplate>
+    </CollectionView.ItemTemplate>
+</CollectionView>
+```
+
+### Responsive Layouts
+
+Build adaptive UIs that work across different screen sizes:
+
+```xml
+<Grid RowDefinitions="Auto,*" ColumnDefinitions="*,*">
+    <Label Grid.Row="0" Grid.ColumnSpan="2" Text="Header" />
+    <BoxView Grid.Row="1" Grid.Column="0" Color="Red" />
+    <BoxView Grid.Row="1" Grid.Column="1" Color="Blue" />
+</Grid>
+```
+
+## 📚 Documentation and Resources
+
+### Official Documentation
+- [.NET MAUI Documentation](https://learn.microsoft.com/dotnet/maui/) - Complete guide to building apps
+- [API Reference](https://learn.microsoft.com/dotnet/api/?view=net-maui-10.0) - Detailed API documentation
+- [Controls Documentation](https://learn.microsoft.com/dotnet/maui/user-interface/controls/) - All available controls
+- [What's New in .NET 10](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-10) - Latest features and improvements
+
+### Learning Resources
+- [.NET MAUI Samples](https://github.com/dotnet/maui-samples) - Official sample applications
+- [.NET MAUI Workshop](https://github.com/dotnet-presentations/dotnet-maui-workshop) - Hands-on learning workshop
+- [Microsoft Learn](https://learn.microsoft.com/training/paths/build-apps-with-dotnet-maui/) - Free training modules
+- [.NET MAUI Blog](https://devblogs.microsoft.com/dotnet/category/net-maui/) - Latest news and updates
+
+### Community Resources
+- [.NET MAUI Community Toolkit](https://learn.microsoft.com/dotnet/communitytoolkit/maui/) - Additional controls, behaviors, and converters
+- [Awesome .NET MAUI](https://github.com/jsuarezruiz/awesome-dotnet-maui) - Curated list of resources
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/.net-maui) - Community Q&A
+
+## 💬 Feedback and Support
+
+We welcome your feedback and contributions!
+
+### Getting Help
+- **Documentation**: Check the [official documentation](https://learn.microsoft.com/dotnet/maui/)
+- **Q&A**: Ask questions on [Stack Overflow](https://stackoverflow.com/questions/tagged/.net-maui) with the `.net-maui` tag
+- **Discussions**: Join [GitHub Discussions](https://github.com/dotnet/maui/discussions) for community conversations
+
+### Reporting Issues
+- **Bug Reports**: File bugs at [GitHub Issues](https://github.com/dotnet/maui/issues)
+- **Feature Requests**: Suggest new features in [GitHub Issues](https://github.com/dotnet/maui/issues)
+- **Security Issues**: Report security vulnerabilities via [Microsoft Security Response Center](https://msrc.microsoft.com/)
+
+### Community
+- **Discord**: Join the [.NET Discord server](https://aka.ms/dotnet-discord)
```

Comment:
Lets also mention the MAUIverse Discord at https://mauiverse.net/discord, mention that this is a community driven server, not Microsoft official